code/game/objects/structures/signs/sign_eyechart.dm

```diff
@@ -0,0 +1,49 @@
+/obj/structure/sign/eyechart
+	icon_state = "eyechart"
+	name = "eye chart"
+	desc = "A poster with a series of letters in different sizes, used to test blindness - I mean, vision."
+	is_editable = TRUE
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/eyechart, 32)
+
+/obj/structure/sign/eyechart/examine(mob/user)
+	. = ..()
+	if(isobserver(user))
+		return
+
+	var/report = ""
+	if(!user.can_read(src, READING_CHECK_LITERACY, silent = TRUE) || !user.has_language(/datum/language/common, UNDERSTOOD_LANGUAGE))
+		if(user.is_blind())
+			return // you are hopeless
+		report = span_warning("You don't actually know what any of the symbols mean.")
+
+	else if(user.is_blind())
+		report = span_notice("Yes, feels like... \"E, P, D...\" Good thing this chart has braille!")
```

Comment:
Doesn't actually check if you have any arms.

Reply:
You can't examine w/o arms so it's asserted